getting-started-command-mode/src/main/docker/Dockerfile.native

```diff
@@ -0,0 +1,22 @@
+####
```

Comment:
Do we really need dockerfiles for command line?

Reply:
if you want to build a native binary for your linux container when running on windows or osx then you do.

Reply:
right! Make sense... How to work around the lack of cross-compilation! Docker!